source/plain.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+function plain (array){
+	return array.reduce((accumulator, currentValue)=>{
+	if( Array.isArray(currentValue)){
```

Comment:
```suggestion
	if(Array.isArray(currentValue)){
```

А вот тут обычно нет)